packages/connectivity/src/scp-cf/jwt.spec.ts

```diff
@@ -82,17 +82,15 @@ describe('jwt', () => {
 
     afterEach(() => {
       nock.cleanAll();
-      verificationKeyCache.clear();
       delete process.env.VCAP_SERVICES;
     });
 
     it('succeeds and decodes for correct key', async () => {
-      nock(jku).get('/').reply(200, responseWithPublicKey());
-
+      nock(jku).get('/').times(999).reply(200, responseWithPublicKey());
       await expect(
         verifyJwt(signedJwtForVerification(jwtPayload, jku))
       ).resolves.toEqual(jwtPayload);
-    });
+    }, 600000);
```

Comment:
Is this intended to stay? Running it 999 times may slow down tests a bit much, is it not?